generate-changeset.mjs

```diff
@@ -95,18 +97,21 @@ log('Building on this branch');
 await runCommand('npm', ['run', 'build']);
 log('Parsing out differences');
 
-const [mainRules, mainConfig] = await loadConfig(dir);
-const [branchRules, branchConfig] = await loadConfig(process.cwd());
+const mainConf = await loadConfig(dir);
+const branchConf = await loadConfig(process.cwd());
+
+const mainRules = mainConf.rules || {};
+const branchRules = mainConf.rules || {};
```

Comment:
Typo? "Conf" makes me think of a conference. I prefer the explicit "Config" verbiage if possible. 🙂 
```suggestion
const mainConfig = await loadConfig(dir);
const branchConfig = await loadConfig(process.cwd());

const mainRules = mainConfig.rules || {};
const branchRules = mainConfig.rules || {};
```

Reply:
Oh yeah I renamed it so TS would make me fix all the places where I was using the old variable. I meant to rename it back but I forgot, I'll do that now